libhb/common.c

```diff
@@ -5878,6 +5913,13 @@ static int pix_fmt_is_supported(hb_job_t * job, int pix_fmt)
                 {
                     return 0;
                 }
+            case HB_FILTER_RENDER_SUB:
```

Comment:
I don't think this does what you intended.  E.g. If pix_fmt is AV_PIX_FMT_YUV420P10, it will return 0 because the first condition is true.